lib/generated_plugin_registrant.dart

```diff
@@ -0,0 +1,30 @@
+//
+// Generated file. Do not edit.
```

Comment:
I think this file should not be pushed since it is a generated one. Make sure to confirm with the rest of the team